cmake/CMakeDependencies.cmake

```diff
@@ -10,6 +10,7 @@ set(CMAKE_PREFIX_PATH ${CMAKE_PREFIX_PATH} "${CMAKE_SOURCE_DIR}/win32-deps")
 # find JSONCPP
 # TODO: handle windows debug libraries!
 # TODO: fix FindJSONCPP file!
+hunter_add_package(jsoncpp)
 find_package(Jsoncpp)
```

Comment:
> I know, but then I will have to enforce usage of imported libs

You have to use imported libs anyway if you want to create relocatable package.

> This project uses its own Find scripts, so I modified then in the way to find the hunter packages.

It doesn't respect dependencies as far as I understand. Please check that it will find and link ZLIB from Hunter for example (not ZLIB from system).